content/installation/installing-manager.md

```diff
@@ -23,10 +23,10 @@ You can install the [Cloudify CLI]({{< relref "installation/installing-cli.md" >
 
 _To install Cloudify Manager:_
 
-1. Go to the download page of the [Cloudify website](http://cloudify.co/download/) and download the Cloudify Manager RPM file.
-1. Copy the RPM file to your target host.
-1. From the CLI of your target host, run: ```sudo yum install <RPM file path>```
-   For example: ```sudo yum install /etc/home/cloudify-manager-install-4.3ga.x86_64.rpm```
+1. Go to the download page on the [Cloudify website](http://cloudify.co/download/) and download the Cloudify Manager RPM file.
+1. Copy the RPM file to your target hostֿ (or download it directly to the host, using a tool like `curl` or `wget`).
```

Comment:
We're not allowing direct download because we need people to fill in the marketing form before they download.

Reply:
Fair enough